### PR TITLE
closes #2 Use redis-py instead of aioredis to support redis-py 4+

### LINKED ISSUE #2: use redis 4 instead of aioredis
The aioredis code has now been migrated into redis for a while now. It is probably time to switch.

### PR DESCRIPTION
**Pull request by rmartin-federated**
Fix for #2

### CODE REVIEW

File: quart_redis/__init__.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import logging
 
-from aioredis import Redis, from_url
+from redis import asyncio as Redis
+from redis import from_url
```
Comment (enchant97) [inline quart_redis/__init__.py]: Looking at the docs should this not be:

```python
from redis.asyncio import Redis, from_url
```

File: quart_redis/__init__.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 from quart import Quart
 
 __all__ = ["RedisHandler", "get_redis"]
 __version__ = "1.0.0"
 
 
 class RedisHandler:
+    print('Our Fork')
```
Comment (enchant97) [inline quart_redis/__init__.py]: I don't think this should be here

File: quart_redis/__init__.py [hunk 3]
```diff
@@ -46,7 +48,7 @@ async def init_redis():
 
         @app.after_serving
         async def close_redis():
-            await RedisHandler._connection.close()
+            RedisHandler._connection.close()
```
Comment (enchant97) [inline quart_redis/__init__.py]: The docs say it's an awaitable: [example](https://redis-py.readthedocs.io/en/stable/examples/asyncio_examples.html#Connecting-and-Disconnecting)

**Review by enchant97 — CHANGES_REQUESTED** [review]
Thanks for current work I haven't had time to migrate it to redis 4, however I think there needs to be some changes before it can be merged.

Also the pip requirement `aioredis ~= 2.0` in setup.cfg will need changing to something like: `redis >= 4.2, < 5`.

**Comment by ltsar-federated** [comment]
So we are encountering an issue where the "Redis" object returned from `get_redis` is having it's namespace clobbered and refusing to be `await`ed. We are going to use explicit typing for the return types from `get_redis` like `-> aioredis.Redis`

**Comment by ltsar-federated** [comment]
Actually it seems like PyCharm is just wrong, code seems to work fine with the awaits. Pushing new commit shortly. Thanks for the head start!

File: quart_redis/__init__.py (hunk 1 above)
Comment (ltsar-federated) [reply quart_redis/__init__.py]: Got it in latest commit
@rmartin-federated

**Comment by ltsar-federated** [comment]
Newest push should be good. Thanks for the speedy review! @rmartin-federated

File: quart_redis/__init__.py [hunk 4]
```diff
@@ -54,6 +54,7 @@ def get_connection(cls) -> Redis:
         """
         get the shared redis connection
         """
+        print(f'Type Returned: {type(cls._connection)}')
```
Comment (enchant97) [inline quart_redis/__init__.py]: Remove this, otherwise it will pollute people's stdout.

**Review by enchant97 — CHANGES_REQUESTED** [review]

**Review by enchant97 — APPROVED** [review]

**Pull request merged**